samples/hello-world/index.v6.js

```diff
@@ -0,0 +1,84 @@
+/*eslint node/no-unsupported-features: ["error", {version: 6}]*/
```

Comment:
A comment similar to the index.js comment that was requested would be nice.

Reply:
Done